### ISSUE TITLE
Target/Budget configuration confusing

### ISSUE THREAD

**Issue by brancz** [metalmatze/slo-libsonnet#45]
There are multiple ways to specify a target/budget:

* `latencyburn` has `latencyTarget` and `latencyBudget`, target being the threshold of "too high duration", budget being the 1-<availability target> value.
* `errorburn` has `target` being the targeted availability like `99.99` (and a deprecated `errorBudget`, that works like `latencyBudget`.
* `errorlatencyburn` has `target` which is like the non-deprecated `target` of `errorburn`.

It would be great if all of these could be consistent in their language and use. I'd be more than happy to make the changes once agreed on the details.

I would propose:

Use `target` everywhere to specify the availability target, as in `99.99` for example. And rename `latencyTarget` to `latencyThreshold`. Wdyt?

@metalmatze

**Comment by metalmatze** [comment]
Yes, these have been in a limbo state where we were migrating from one to the other. 
`target` should be the way forward, as can be seen here:
https://github.com/metalmatze/slo-libsonnet/blob/master/slo-libsonnet/error-burn.libsonnet#L12-L16
